### PR TITLE
fix some definitions

### PR DESCRIPTION
**Pull request by fancyweb**

### CODE REVIEW

File: oas_apivideo.json [hunk 1]
```diff
@@ -1429,12 +1429,36 @@
...
+        ],
         "parameters": [
           {
             "name": "token",
             "in": "query",
             "required": false,
             "type": "string"
+          },
+          {
+            "name": "file",
```
Comment (fancyweb) [inline oas_apivideo.json:1443]: This endpoint is missing its form data and the content-range header.

File: oas_apivideo.json [hunk 2]
```diff
@@ -1516,6 +1540,17 @@
             "required": false,
             "type": "string"
           },
+          {
+            "name": "sortOrder",
```
Comment (fancyweb) [inline oas_apivideo.json:1544]: The get all live streams is missing this query param.

File: oas_apivideo.json [hunk 3]
```diff
@@ -3675,18 +3710,17 @@
...
       }
     }
   },
   "securityDefinitions": {
-    "oauth2": {
-      "type": "oauth2",
-      "authorizationUrl": "https://api.video",
-      "flow": "implicit",
-      "scopes": {}
+    "bearerAuth": {
```
Comment (fancyweb) [inline oas_apivideo.json:3720]: The API auth method is a JWT.

File: oas_apivideo.json (hunk 1 above)
Comment (dougsillars) [reply oas_apivideo.json:1443]: This *may* crash readme.  but lets add it, and then see :)

**Review by dougsillars — APPROVED** [review]
LGTM

**Pull request merged**